table/block_based_table_builder.cc

```diff
@@ -618,7 +247,8 @@ struct BlockBasedTableBuilder::Rep {
   TableProperties props;
 
   bool closed = false;  // Either Finish() or Abandon() has been called.
-  std::unique_ptr<FilterBlockBuilder> filter_block;
+  std::unique_ptr<FilterBlockBuilder> filter_block_gc;
+  FilterBlockBuilder* filter_block;
```

Comment:
That was not the name chosen by me. Would like to do this refactoring as part of this patch?